src/src/WebServer/Metrics.cpp

```diff
@@ -1,136 +1,138 @@
+  addHtml(F("load device percentage load\n"));
+  addHtml(prefixTYPE);
+  addHtml(F("load gauge\n"));
+  addHtml(F("espeasy_load "));
+  addHtml(getValue(LabelType::LOAD_PCT));
+  addHtml('\n');
+
+  // Free RAM
+  addHtml(prefixHELP);
+  addHtml(F("free_ram device amount of RAM free in Bytes\n"));
+  addHtml(prefixTYPE);
+  addHtml(F("free_ram gauge\n"));
+  addHtml(F("espeasy_free_ram "));
+  addHtml(getValue(LabelType::FREE_MEM));
+  addHtml('\n');
+
+  // Free RAM
+  addHtml(prefixHELP);
+  addHtml(F("free_stack device amount of Stack free in Bytes\n"));
+  addHtml(prefixTYPE);
+  addHtml(F("free_stack gauge\n"));
+  addHtml(F("espeasy_free_stack "));
+  addHtml(getValue(LabelType::FREE_STACK));
+  addHtml('\n');
+
+  // Wifi strength
+  addHtml(prefixHELP);
+  addHtml(F("wifi_rssi Wifi connection Strength\n"));
+  addHtml(prefixTYPE);
+  addHtml(F("wifi_rssi gauge\n"));
+  addHtml(F("espeasy_wifi_rssi "));
+  addHtml(getValue(LabelType::WIFI_RSSI));
+  addHtml('\n');
+
+  // Wifi uptime
+  addHtml(prefixHELP);
+  addHtml(F("wifi_connected Time wifi has been connected in milliseconds\n"));
+  addHtml(prefixTYPE);
+  addHtml(F("wifi_connected counter\n"));
+  addHtml(F("espeasy_wifi_connected "));
+  addHtml(getValue(LabelType::CONNECTED_MSEC));
+  addHtml('\n');
+
+  // Wifi reconnects
+  addHtml(prefixHELP);
+  addHtml(F("wifi_reconnects Number of times Wifi has reconnected since boot\n"));
+  addHtml(prefixTYPE);
+  addHtml(F("wifi_reconnects counter\n"));
+  addHtml(F("espeasy_wifi_reconnects "));
+  addHtml(getValue(LabelType::NUMBER_RECONNECTS));
+  addHtml('\n');
+
+  // devices
+  handle_metrics_devices();
+
+  TXBuffer.endStream();
 }
 
-void handle_metrics_devices(){
-    for (taskIndex_t x = 0; validTaskIndex(x); x++)
-    {        
-        const deviceIndex_t DeviceIndex = getDeviceIndex_from_TaskIndex(x);
-        const bool pluginID_set         = INVALID_PLUGIN_ID != Settings.TaskDeviceNumber[x];
-         if (pluginID_set){
-            if (Settings.TaskDeviceEnabled[x]){
-                String deviceName = getTaskDeviceName(x);
-                if (deviceName.isEmpty()) { // Empty name, then use taskN
-                    deviceName = F("task");
-                    deviceName += x;
-                }
-                addHtml(F("# HELP espeasy_device_"));
-                addHtml(deviceName);
-                addHtml(F(" Values from connected device\n"));
-                addHtml(F("# TYPE espeasy_device_"));
+void handle_metrics_devices() {
+  for (taskIndex_t x = 0; validTaskIndex(x); x++) {
+    const deviceIndex_t DeviceIndex = getDeviceIndex_from_TaskIndex(x);
+    const bool pluginID_set         = INVALID_PLUGIN_ID != Settings.TaskDeviceNumber[x];
+
+    if (pluginID_set) {
+      if (Settings.TaskDeviceEnabled[x]) {
+        String deviceName = getTaskDeviceName(x);
+
+        if (deviceName.isEmpty()) { // Empty name, then use taskN
+          deviceName  = F("task");
+          deviceName += x + 1;
```

Comment:
Is this the only change? (apart from the uncrustify)

Reply:
Yep, and some string-deduplications, saving around 75 bytes of .bin size on ESP8266 builds.